executorlib/backend/interactive_parallel.py

```
@@ -97,7 +97,23 @@ def main() -> None:
             and "args" in input_dict
             and "kwargs" in input_dict
         ):
-            memory.update(call_funct(input_dict=input_dict, funct=None, memory=memory))
+            try:
+                memory.update(
+                    call_funct(input_dict=input_dict, funct=None, memory=memory)
+                )
+            except Exception as error:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)